client/protocols/guidance.py

```diff
@@ -0,0 +1,178 @@
+from client.protocols.common import models
```

Comment:
Please add a docstring at the top of this module to explain what this is used for. Some questions worth answering:

* What server does it communicate to, and why?
* What are treatment groups used for?
* How does this affect the behavior of OK?